src/adapters/marketplace/v00.js

```diff
@@ -100,41 +100,30 @@ class V00_MarkeplaceAdapter {
   }
 
   async makeOffer(listingId, ipfsBytes, data, confirmationCallback) {
-    const { affiliate, arbitrator, commission = {}, finalizes, totalPrice = {}, unitsPurchased } = data
+    const { affiliate, arbitrator, commission, finalizes, totalPrice = {}, unitsPurchased } = data
```

Comment:
affiliate, arbitrator, finalizes will get saved in IPFS. We had discussed storing the blockchain data under a separate "_untrustworthy" object. So that developers don't get confused as to what is the source of truth between onchain and offchain data. Add a TODO here so that we remember ?

Reply:
👍